extension.Rmd

```diff
@@ -0,0 +1,175 @@
+library(stargazer)
+library(grid)
+library(gridExtra)
+
+data <- read.dta("nonimp.dta")
+
+# Cleaning up data
+data <- rename(data, ext_vote = rexvote)
+data <- rename(data, age_1 = age1) # (18–29 years)
+data <- rename(data, age_2 = age2) # (30–45 years)
+data <- rename(data, age_4 = age4) # (>65 years)
+data <- rename(data, edu_1 = mye1) # Education: middle/high
+data <- rename(data, edu_2 = mye2) # Education: university
+data <- rename(data, farmer_own = farmerown) 
+data <- rename(data, left_right_scale = zlrs) # Left-Right Scale
+data <- rename(data, eu_neg = euschlecht) # Negative evaluation of EU membership of one's own country
+data <- rename(data, z_dem_satis = zsatisdmo) # Dissatisfied: Democracy
+data <- rename(data, dispro_elect = disp) # Disproportionality
+data <- rename(data, federalism = lfed1) # Decentralization
+data <- rename(data, z_asylumseeker = zasylumseekers) 
+data <- rename(data, z_unemp = zsur) # Unemployment rate
+data <- rename(data, z_replacement = zreplacementrate) # Unemp benefits
+data <- rename(data, max_er = rmax) # Maximum toughness in extreme right discourse
+
+```
+
+
+# Preparing data for analysis
+
+```{r}
+# Create centered variables for Saliance and Variance based on grand mean centering according to author
+
+data <- data  %>% 
+  mutate(salience_mean_c = (salienzmean - 3.84568)) %>% 
+  mutate(var_c = rvar - 21.75423)
+
+# Make male variable numeric
+data <- data %>%
+  mutate(male = as.numeric(male))
+
+# Create a dummy variable for country (transform from character to factor variable)
+data <- data %>%
+  mutate(country = factor(sortcountry, labels = c("AT", "BE","DE-E","DE-W","DK","ES","FI","FR","GR","IT","LU","NL","NO","PT","SE")))
+```
+
+
+# Model 1: Adorno Thesis (Personality Traits and Value Orientations)
+The first model is based on Theodor Adorno's 1950 theory that support for extreme right parties (in his case, fascist parties) originates from supporters' personality traits and value orientations, which make individuals more receptive to the appeals of extreme right parties.  We operationalize "personality traits and value orientations" using the individual level characteristics from our data set: gender, age, occupation, marital status, and self-placement on a left-right scale. 
+
+```{r}
+library(separationplot)
+library(Hmisc) 
+
+model_1 <- glm(ext_vote ~ male + age_1 + age_2 + age_4 + edu_1 + edu_2 + farmer_own + worker + retired + unemployed + partner + left_right_scale + country - 1, data = data)
+summary(model_1)
+actual <- model_1$y # Create vector with actual values for vote intention for extreme right parties
+
+log.odds <- model_1$fitted.values # Create a vector with log-odd values
+# Create link function to transform log-odds into predicted probabilities
+link <- function(P) {
+  1/(1+exp(-P))
+} 
+pred <- link(log.odds) # Transform log-odds into predicted probabilities
+pred <- as.vector(pred) 
+range(pred) # variation in the predicted probabilities: very little variation.
```

Comment:
this might have been a good graph.  Even with all those starts there is really very little variation in predictions